### PR TITLE
42). Add topic files 131 to 134

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also fixes some article files.

Please merge #18439, #18541, #18580, #18589, #18673, #18700, #18711, and #18781 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

File: docs/standard/linq/use-xslt-transform-xml-tree.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+description: Learn to use XSLT to transform an XML tree, using XmlReader to read and XmlWriter to write.
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 373a2699-d4c5-471b-9bda-c1f0ab73b477
+---
+# Use XSLT to transform an XML tree (LINQ to XML)
+
+You can use XSLT to transform an XML tree, using <xref:System.Xml.XmlReader> to read and <xref:System.Xml.XmlWriter> to write.
```
Comment (mairaw) [inline docs/standard/linq/use-xslt-transform-xml-tree.md:12]: This should fix the linting error

```suggestion
You can use XSLT to transform an XML tree, using <xref:System.Xml.XmlReader> to read and <xref:System.Xml.XmlWriter> to write.
```

**Comment by mairaw** [comment]
@jameshkramer we can resolve the merge conflicts here now

**Comment by jameshkramer** [comment]
@mairaw 

The conflicts are resolved - Jim

**Review by mairaw — APPROVED** [review]
Looks great. Thanks @jameshkramer!

**Pull request merged**